src/frontend/src/pages/DesignReviewView.tsx

```diff
@@ -0,0 +1,186 @@
+
+const DRCView: React.FC<DRCViewProps> = ({ usersToAvailabilities, iconData }) => {
+  const totalUsers = usersToAvailabilities.size;
+  const availableUsers = new Map<number, User[]>();
+  const unavailableUsers = new Map<number, User[]>();
+  const numberOfTimeSlots = times.length * daysOfWeek.length;
+  const [currentAvailableUsers, setCurrentAvailableUsers] = useState<User[]>([]);
+  const [currentUnavailableUsers, setCurrentUnavailableUsers] = useState<User[]>([]);
+
+  function createAvailableUsers() {
+    for (let time = 0; time < numberOfTimeSlots; time++) {
+      availableUsers.set(time, []);
+    }
+
+    usersToAvailabilities.forEach((availableTimes, user) => {
+      availableTimes.forEach((time) => {
+        const usersAtTime = availableUsers.get(time) || [];
+        usersAtTime.push(user);
+        availableUsers.set(time, usersAtTime);
+      });
+    });
+    return availableUsers;
+  }
+
+  function createUnavailableUsers() {
+    const allUsers = [...usersToAvailabilities.keys()];
+    for (let time = 0; time < numberOfTimeSlots; time++) {
+      const currentUsers = availableUsers.get(time) || [];
+      const currentUnavailableUsers = allUsers.filter((user) => !currentUsers.includes(user));
+      unavailableUsers.set(time, currentUnavailableUsers);
+    }
+    return unavailableUsers;
+  }
+
+  const handleOnMouseOver = (index: number) => {
+    setCurrentAvailableUsers(availableUsers.get(index) || []);
+    setCurrentUnavailableUsers(unavailableUsers.get(index) || []);
+  };
+
+  const handleOnMouseLeave = () => {
+    setCurrentAvailableUsers([]);
+    setCurrentUnavailableUsers([]);
+  };
+
+  const renderDayHeaders = () => {
+    return [
+      <TimeSlot backgroundColor="#D9D9D9" isModal={false} />,
+      daysOfWeek.map((day) => <TimeSlot key={day} backgroundColor="#D9D9D9" text={day} fontSize={'1em'} isModal={false} />)
+    ];
+  };
+
+  const renderSchedule = () => {
+    createAvailableUsers();
+    createUnavailableUsers();
+    return (
+      <Grid container>
+        {renderDayHeaders()}
+        {times.map((time, timeIndex) => (
+          <Grid container item xs={12} onMouseLeave={handleOnMouseLeave}>
+            <TimeSlot backgroundColor="#D9D9D9" text={time} fontSize={'1em'} isModal={false} />
+            {daysOfWeek.map((_day, dayIndex) => {
+              const index = dayIndex * times.length + timeIndex;
+              return (
+                <TimeSlot
+                  key={index}
+                  backgroundColor={getBackgroundColor(availableUsers.get(index)?.length, totalUsers)}
+                  onMouseOver={() => handleOnMouseOver(index)}
+                  isModal={false}
+                  icon={iconData.get(index)}
+                />
+              );
+            })}
+          </Grid>
+        ))}
+      </Grid>
+    );
+  };
+
+  const renderLegend = () => {
+    const colors = ['#D9D9D9', '#E0C0C1', '#E89A9B', '#E4797A', '#EF4345', '#D70C0F'];
```

Comment:
if this is also used in the schedule view, it should be a const in the utils file: `AVAILABILITY_SCHEDULE_COLORS`